dev-libs/libp11/libp11-0.4.3.recipe

```diff
@@ -0,0 +1,71 @@
+SUMMARY="A library that implements a layer on top of PKCS11 API"
+DESCRIPTION="The official header files for PKCS11 Standard were released \
+under a non-free license. libp11 does not include these header files, but \
+instead uses a rewritten pkcs11.h header file from the Scute project."
+HOMEPAGE="https://github.com/opensc/libp11"
+COPYRIGHT="1991, 1999 Free Software Foundation"
+LICENSE="GNU LGPL v2"
+REVISION="1"
+SOURCE_URI="$HOMEPAGE/releases/download/libp11-$portVersion/libp11-$portVersion.tar.gz"
+CHECKSUM_SHA256="938bf73fc57ef8b543c6a2ceab617d7d83ccf60456241b73a61adb5ca37cd7c5"
+SOURCE_DIR="libp11-$portVersion"
+
+ARCHITECTURES="x86_gcc2 ?x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	libp11$secondaryArchSuffix = $portVersion
+	lib:libp11$secondaryArchSuffix = $portVersion
+	lib:libpkcs11$secondaryArchSuffix = $portVersion
+	lib:pkcs11$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libcrypto$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libp11${secondaryArchSuffix}_devel = $portVersion
+	devel:libp11$secondaryArchSuffix = $portVersion
+	devel:libpkcs11$secondaryArchSuffix = $portVersion
+	devel:pkcs11$secondaryArchSuffix = $portVersion
+	"
+REQUIRES_devel="
+	haiku${secondaryArchSuffix}_devel
+	libp11$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libcrypto$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:awk
+	cmd:diff
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:gcc$secondaryArchSuffix
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	runConfigure ./configure
```

Comment:
Should be
```
	runConfigure ./configure --with-enginesdir=$libDir/engines
```
otherwise "make install" tries to copy pkcs11.so in the $libDir/engines/ directory of the openssl package.
But that directory is read-only, so we need to tell libp11 to install its pkcs11.so in its own $libDir/engines/.